queryfilter/textfilters.py

```diff
@@ -9,37 +9,37 @@
 @add_metaclass(abc.ABCMeta)
 class TextMatchMixin(object):
     @abc.abstractmethod
-    def _is_value_matched(self, value): pass
+    def is_value_matched(self, value): pass
```

Comment:
It's ok to have it with prefix `_` here tho XD since it's under `class` instead of `def`